app/controllers/groups_controller.rb

```diff
@@ -194,15 +194,17 @@ def posts
     group = find_group(:group_id)
     guardian.ensure_can_see_group_members!(group)
 
-    posts = group.posts_for(guardian, params.permit(:before_post_id, :category_id)).limit(20)
+    posts =
+      group.posts_for(guardian, params.permit(:before_post_id, :before, :category_id)).limit(20)
```

Comment:
Is the `before_post_id` param being used by something else in the frontend? It would be an API change to remove, but I am curious to know if we're still consuming it.

Reply:
I couldn't find anything else using this param, I just kept it to be on the safe side/avoid a breaking change in the API.

I guess I was extra cautious here because I recently had to ["undo"](https://github.com/discourse/discourse-calendar/commit/b9188c4c162e2d8a3eee206c2deeca436fc1e4a7) an API change because of external/hidden uses from customers.